packages/libs/hd-wallet/src/SLIP10/utils/sign.ts

```diff
@@ -10,15 +10,18 @@ export interface ISignatureWithPublicKey {
 
 /**
  * Derive a key pair using a seed and an index.
- * @param {Uint8Array} seed - The seed for key derivation.
+ * @param {ArrayBuffer} seed - The seed for key derivation.
  * @param {number} index - The index for key derivation.
  * @returns {{ privateKey: string; publicKey: string }} - Returns the derived private and public keys.
  */
 export const deriveKeyPair = (
-  seed: Uint8Array,
+  seed: ArrayBuffer,
```

Comment:
Why are you moving from Uint8Array to ArrayBuffer?

Reply:
I didn't move from Uint8Array, it is just a view on to of ArrayBuffer and actually it just extends the class.
So you can also pass Uint8Array or even Buffer